client/my-sites/upgrades/checkout/style.scss

```diff
@@ -2,15 +2,20 @@
 	position: relative;
 
 	.payment-box {
-		height: 0;
 		margin-bottom: 0;
-		opacity: 0;
-		overflow: hidden;
 		padding: 0;
-		transform: translateZ(0) scale(.8); // Zoom out effect
-		transition: all .3s ease-in-out;
-		visibility: hidden; // To deal with layering issues
 		width: 100%;
+		overflow: hidden;
+
+		height: 0;
+		opacity: 0;
+		visibility: hidden;
+		transform: translateZ(0) scale(.8); // Zoom out effect
```

Comment:
Did you intend to animate this as well? The scale(.8) sticks to the payment-box with your changes.